engine/samples/hello-cubos/main.cpp

```diff
@@ -0,0 +1,48 @@
+/// [Include]
+#include <cubos/engine/cubos.hpp>
+
+using cubos::engine::Cubos;
+/// [Include]
+
+/// [Hello Cubos]
+void sayHelloCubos()
+{
+    CUBOS_INFO("Hello CUBOS");
+}
+/// [Hello Cubos]
+
+/// [Hello World]
+void sayHello()
+{
+    CUBOS_INFO("Hello");
+}
+
+void sayWorld()
+{
+    CUBOS_INFO("World");
+}
+/// [Hello World]
+
+int main()
+{
+    /// [Engine]
+    Cubos cubos{};
+    /// [Engine]
+
+    /// [Tags]
+    cubos.tag("helloTag").before("worldTag");
+    /// [Tags]
+
+    /// [Set Startup]
+    cubos.startupSystem(sayHelloCubos);
+    /// [Set Startup]
+
+    /// [Set Systems]
+    cubos.system(sayHello).tagged("helloTag");
+    cubos.system(sayWorld).tagged("worldTag");
+    /// [Set Systems]
+
+    /// [Run]
+    cubos.run();
+    /// [Run]
+}
```

Comment:
Include the closing brace in the snippet to make it clear where the program ends.